packages/cos/collection.yaml

```diff
@@ -0,0 +1,9 @@
+packages:
```

Comment:
this is a way to share the same build yaml file, https://luet-lab.github.io/docs/docs/concepts/packages/collections/